.travis.yml

```diff
@@ -46,9 +47,15 @@ script:
 jobs:
   include:
     - stage: sniff
+      addons:
+        apt:
+          packages:
+            - libxml2-utils
       script:
         - composer lint
         - composer phpcs
+        - xmllint --noout --schema ./vendor/squizlabs/php_codesniffer/phpcs.xsd ./*/ruleset.xml
```

Comment:
I think the order should be changed here with the `composer phpcs` one. It makes more sense to me to validate the ruleset first, and only then apply it.

Reply:
Makes sense to me. Fixed.